models/job.go

```diff
@@ -329,7 +329,9 @@ type HTTPDependency struct {
 // JobService provides a high-level operations on DAGs
 type JobService interface {
 	// Create constructs a Job and commits it to a storage
-	Create(context.Context, NamespaceSpec, JobSpec) error
+	Create(context.Context, NamespaceSpec, JobSpec) (*JobSpec, error)
+	// BulkCreate save bulk of JobSpec within same namespace to persistent
+	BulkCreate(context.Context, NamespaceSpec, []JobSpec, progress.Observer) []JobSpec
```

Comment:
Do we need to put it as the contract? As far as I see, it's only being used inside the service (apart from the unit test)